sa/sa_test.go

```diff
@@ -2574,3 +2578,139 @@ func TestIncidentsForSerial(t *testing.T) {
 	test.AssertNotError(t, err, "Failed to retrieve incidents for serial")
 	test.Assert(t, len(incidentsForSerial) == 1, "No active incidents returned")
 }
+
+type inmemSA struct {
+	sapb.StorageAuthorityClient
+	Impl *SQLStorageAuthority
+}
+
+type sfiData struct {
+	serial *sapb.IncidentSerial
+	err    error
+}
+
+type sfiClient struct {
+	// grpc.ClientStream is embedded to meet the
+	// StorageAuthority_SerialsForIncidentClient interface.
```

Comment:
Might also want to mention that we expect this to be nil, and calling methods on it will result in a panic; but that's okay because we don't expect our tests to call any methods on it.